Update _index.md (#585)
EGI-Federation/documentation: content/en/users/compute/content-distribution/_index.md

In the email, include the following information:

- Name of the VO or CVMFS repository.

> - Your voperson_id from EGI CheckIn.

Reviewer: ```suggestion
- Your `voperson_id` from EGI Check-in.
```

> - Your `voperson_id` from EGI CheckIn.

Reviewer: ```suggestion
- Your `voperson_id` from EGI Check-in.
```
Let's use the proper service name.

> - Your `voperson_id` from EGI Check-in.

Reviewer 2: i think it's better to add instruction on how to get the voperson_id or point to some existing docs

Reviewer: Or maybe in fact it's just easier to replace `voperson_id` by Check-in ID, keeping the same link?

Reviewer 2: yes they are the same:-)

Reviewer: OK, I've updated this.

## Distributing new content

> To login to the service, just ssh:  

Reviewer: ```suggestion
To login to the service, just use `ssh`:  
```

```shell

> $  ssh @cvmfs-upload01.gridpp.rl.ac.uk

Reviewer 3: ```suggestion
$  ssh cvmfs-upload01.gridpp.rl.ac.uk
```

Author: Only method currently provided is interactive login.
Web access is not [yet] implemented. 
I know some people have a crontab in their repository to git pull periodically. That is OK, but needs to be requested and approved by our security team.

Reviewer 3: I still see the `@` in the command line there, this fails to run

```

If you are the Content Manager for more than one repository, you would need to

> specify explicit which account you want to login to. This account is built as <reponame>+"sgm". 

Reviewer: ```suggestion
specify explicit which account you want to login to. This account is built as `$REPONAME+"sgm"`. 
```

For example, for repository **dirac.egi.eu**, UNIX account is **diracsgm**.

```shell

> $ ssh <myreposgm>@cvmfs-upload01.gridpp.rl.ac.uk

Reviewer: ```suggestion
$ ssh $MYREPOSGM@cvmfs-upload01.gridpp.rl.ac.uk
```

```

To copy data:

```shell

> $ scp <source> cvmfs-upload01.gridpp.rl.ac.uk:<destination>

Reviewer: ```suggestion
$ scp $SOURCE cvmfs-upload01.gridpp.rl.ac.uk:$DESTINATION
```

Reviewer: `<>` are understood as inline HTML tags

```

When running the ssh/scp commands, a message like this is displayed:
 
```shell
[neo@matrix ~]# ssh gridppsgm@cvmfs-upload01.gridpp.rl.ac.uk
Authenticate at
-----------------

> https://aai.egi.eu/device?user_code=IITC-KKUP

Reviewer: Can you please add something like`https://aai.egi.eu/device*` to https://github.com/EGI-Federation/documentation/blob/main/.github/linters/mlc_config.json ?
It would also probably be better to use a fake user code, this one seems to allow me to use it.

Author: will do

## Distributing new content

To login to the service, just use `ssh`:  

```shell
$  ssh cvmfs-upload01.gridpp.rl.ac.uk
```

If you are the Content Manager for more than one repository, you would need to

> specify explicitly which account you want to login to. This account is built as `\<reponame\>+"sgm"`.

Reviewer: ```suggestion
specify explicitly which account you want to login to. This account is built as `reponame+"sgm"`.
```

> For example, for repository **dirac.egi.eu**, UNIX account is **diracsgm**.

Reviewer: ```suggestion
For example, for repository `dirac.egi.eu`, UNIX account is `diracsgm`.
```

```shell

> $ ssh \<myreposgm\>@cvmfs-upload01.gridpp.rl.ac.uk

Reviewer: ```suggestion
# Replace with the proper username
$ ssh diracsgm@cvmfs-upload01.gridpp.rl.ac.uk
```

```

To copy data:

```shell
$ scp \<source\> cvmfs-upload01.gridpp.rl.ac.uk:<destination>
```

When running the ssh/scp commands, a message like this is displayed:

```shell

> [neo@matrix ~]# ssh gridppsgm@cvmfs-upload01.gridpp.rl.ac.uk

Reviewer: ```suggestion
$ ssh gridppsgm@cvmfs-upload01.gridpp.rl.ac.uk
```

## Distributing new content

To login to the service, just use `ssh`:  

```shell
$  ssh cvmfs-upload01.gridpp.rl.ac.uk
```

If you are the Content Manager for more than one repository, you would need to
specify explicitly which account you want to login to. This account is built as `reponame+"sgm"`.
For example, for repository `dirac.egi.eu`, UNIX account is `diracsgm`.

```shell
# Replace with the proper username
$ ssh diracsgm@cvmfs-upload01.gridpp.rl.ac.uk
```

To copy data:

```shell

> $ scp source_file.txt cvmfs-upload01.gridpp.rl.ac.uk:destination_folder/

Reviewer: ```suggestion
# Replace with the proper username
$ scp source_file.txt diracsgm@cvmfs-upload01.gridpp.rl.ac.uk:destination_folder/
```
